.circleci/config.yml

```diff
@@ -137,6 +137,14 @@ jobs:
           name: Type check
           command: yarn test:types
 
+  "Test: unit":
```

Comment:
FYI future notice: use the community orb for easier circle management: https://github.com/react-native-community/react-native-circleci-orb